EIPS/eip-2794.md

```diff
@@ -0,0 +1,121 @@
+- If a validator is forced to exit he/she will no longer be qualified to receive future honor tokens. However he/she can still keep the honor tokens he/she has had and his/her possessed tokens are still valid.
+- If a validator is punished but not forced to exit he/she will still be qualified to receive future honor tokens. But the future honor tokens will record all the penalties the validator has received and the tokens will be in special formats.
+- As time goes by, honor tokens with special designs for big events such as anniversary celebrations can be issued. For instance an honor token for the tenth anniversary of ETH 2.0 Staking can be issued. This is cool and significantly meaningful.
+- As a validator serves longer with good service i.e. no penalties, the honor tokens he/she receives will bring him/her more pride and a stronger sense of community. And these tokens will be more valuable. This will motivate validators to serve with more passion and loyalty while will greatly minimize possibilities of misbehaving. It will significantly prevent mass exit of validators and enhance Ethereum stability and reliability.
+- An honor token is priced in NDAO, and can only be purchased and sold in NDAO and ETH ---- this is a second but not the last application of NDAO. By recalling blockchain's history (e.g. development of DeFi) we can see that blockchain's openness will continuously stimulate innovations and applications.
+- An honor token can by purchased and sold on exchanges such as OpenSea as easy as with just one click. This can be easily implemented with support from these exchanges.
+- Honor tokens will bring both value and wealth to validators, and will very likely be rarities and collectibles. In our proposal when validators exit they will receive NDAO rather than ETH. In this case if ETH's price is higher than it used to be, validators may feel unfair and honor tokens will be a good compensation.
+- Honor tokens are good incentives to attrace validators to join staking as early as possible because these early birds may win unique token with their service time which may be the longest among all the service times recorded in all honor tokens. This is the first ever means to reward an Ethereum's contributor with special honor and will motivate future thoughts in blockchain's development.
+
+From our perspective, the above measures will eliminate the risks of ETH 2.0 staking being harmed by ETH price fluctuations.
+
+## NDAO and ETH Forging
```

Comment:
This is not one of the template sections found in https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md.  While technically additional template sections are allowed, they are weakly discouraged.  Recommend including this into one of the template sections.